### PR TITLE
Firebase ML Kit List Models API implementation

### PR DESCRIPTION
**Pull request by ifielker**
Includes ListModelsPage and ModelIterator

### CODE REVIEW

File: firebase_admin/mlkit.py [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
 
 _MLKIT_ATTRIBUTE = '_mlkit'
+MAX_PAGE_SIZE = 100
```
Comment (hiranya911) [inline firebase_admin/mlkit.py]: Prefix with `_` to indicate that it's not part of the public API.

File: firebase_admin/mlkit.py [hunk 2]
```diff
@@ -69,7 +76,107 @@ def __eq__(self, other):
...
+        project starting from this page. The iterator will never buffer more than
+        one page of models in memory at a time.
+
+        Returns:
+            iterator: An iterator of Model instances.
+        """
+        return ModelIterator(self)
+
+
+class ModelIterator(object):
```
Comment (hiranya911) [inline firebase_admin/mlkit.py]: Rename to `_ModelIterator`. This doesn't need to be in the public API.

File: tests/test_mlkit.py [hunk 3]
```diff
@@ -33,27 +35,84 @@
...
+MODEL_NAME_3 = 'projects/{0}/models/{1}'.format(PROJECT_ID, MODEL_ID_3)
+DISPLAY_NAME_3 = 'displayName3'
+MODEL_JSON_3 = {
+    'name': MODEL_NAME_3,
+    'displayName': DISPLAY_NAME_3
+}
+MODEL_3 = mlkit.Model(MODEL_JSON_3)
+
 _EMPTY_RESPONSE = json.dumps({})
+_DEFAULT_GET_RESPONSE = json.dumps(MODEL_JSON_1)
```
Comment (hiranya911) [inline tests/test_mlkit.py]: Nit: I see some constants prefixed with `_` and some not. Should we be consistent within the same test file?

File: firebase_admin/mlkit.py (hunk 1 above)
Comment (ifielker) [reply firebase_admin/mlkit.py]: done

File: firebase_admin/mlkit.py (hunk 2 above)
Comment (ifielker) [reply firebase_admin/mlkit.py]: done

File: tests/test_mlkit.py (hunk 3 above)
Comment (ifielker) [reply tests/test_mlkit.py]: done

**Pull request merged**